modules/nf-core/gawk/main.nf

```diff
@@ -0,0 +1,35 @@
+process GAWK {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "anaconda::gawk=5.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    tuple val(meta), path("${prefix}")  , emit: output
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
Should we have a better default?
```suggestion
    prefix = task.ext.prefix ?: "${meta.id}.${input.getExtension}"
```
Maybe just a comment that we're looking open to suggestion to improve the default? That way we can merge this in without getting lost in making it perfect.

Reply:
Yeah you're right! This makes more sense :)